doc/radosgw/uadk-accel.rst

```diff
@@ -97,9 +97,9 @@ with hardware accelerators for the HiSilicon Kunpeng platform.
 Make sure all these above kernel configurations are selected.
 
 #. UADK enablement
-If the architecture is aarch64, it will automatically download the UADK source code to build
-the static library. If it runs on other architecture, user can enable it with build parameters
-`-DWITH_UADK=true`
+If the architecture is aarch64, it will automatically download the UADK source
+code to build the static library. If it runs on other architecture, user can
+enable it with build parameters `-DWITH_UADK=true`
```

Comment:
When building on other CPU architectures, the user may enable UADK by adding `-DWITH_UADK=true` to the compilation commandline options.  Note that UADK may not be compatible with all architectures.

Reply:
Building UADK seems like all fine, but the functions work up to hardware compatibility.